src/v0.1.0/properties/structures/_cheminfo_inchikey.yaml

```diff
@@ -12,8 +12,8 @@ type:
   - "string"
   - "null"
 description: |-
-  The standard InChIKey identifier of the structure, as laid out by the InChI Trust (https://www.inchi-trust.org).
-  Standard InChIKey is not guaranteed to be unique and in extremely rare cases the same InChIKey may be assigned to several different structures.
+  An InChIKey identifier of the structure, as laid out by the InChI Trust (https://www.inchi-trust.org).
+  It can be any InChIKey: not necessary a Standard InChIKey, but can also be the latter.
```

Comment:
We should probably also copy the disclaimer here about the InChIKey being potentially non-unique.

Reply:
Copied in 0da02c6c7b18543a27437a80952aec0df88b67b6.